packages/data-models/appConfig.ts

```diff
@@ -43,6 +43,7 @@ const APP_FIELDS = {
   APP_AUTH_USER: `${FIELD_PREFIX}._app_auth_user`,
   APP_SKIN: `${FIELD_PREFIX}._app_skin`,
   APP_THEME: `${FIELD_PREFIX}._app_theme`,
+  HIGHLIGHTED_DATA_ID: `${FIELD_PREFIX}._highlighted_data_id`,
```

Comment:
nit(blocking)
As this (for now) is pretty specific just to the tasks system which is subject to change let's I'd suggest just hardcoding this field into the task.service.ts field and remove from app config. If we really need it to change with skin/theme then you could always add as a suffix to the name. 

In addition we should be more semantic with what the _data_ is referring to, so probably a better name would be along the lines of `_task_highlighted_group_id` or similar